.github/workflows/sorbet.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         if: steps.update.outputs.committed == 'true'
         uses: Homebrew/actions/git-try-push@master
         with:
-          token: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` won't trigger CI.

Reply:
@reitermarkus Thanks! So it's needed for `git push` not not `hub pull-request`, right?

Reply:
@reitermarkus Actually: seems like the other way around as we run CI on `pull_request` and not `branches` (except `master`)

Reply:
I'm not sure, I assume it's needed for `hub pull-request`.